portfolio/src/main/webapp/style.css

```diff
@@ -44,13 +44,14 @@ nav a {
 }
 
 nav a:hover {
-     color: #8B0000;
+    color: #8B0000;
     text-underline-position: below;
 }
 
 h1 {
     text-align: center;
     margin: 100px;
+    nit: alphabetize;
```

Comment:
What I meant was that these declarations should be in alphabetical order (`text-align` should be last). "nit" means that the problem was insignificant so my comment was "nit-picking"